docs/subsystems/typing-indicators.md

```diff
@@ -45,7 +45,7 @@ repeated updates to the server, so that downstream clients know that the
 user is still typing.  (Zulip messages tend to be longer than
 messages in other chat/text clients, so this detail is important.)
 
-We have a small state machine in `static/js/typing_status.js` that
+We have a small state machine in `shared/js/typing_status.js` that
```

Comment:
This change is I think in the wrong commit, since this commit doesn't move that path?

Reply:
Yeah, rebase mistake, fixed.